lib/serial.cpp

```diff
@@ -117,6 +117,7 @@ uint32_t Serial::crc32(uint8_t* buf, int len) {
 }
 
 void Serial::handlePacket(uint8_t* buf, int len) {
+    // FIXME (aw): why is there no implementation here?
```

Comment:
we should make this virtual, probably does not make much sense to have a default implementation

Reply:
yes, I guess this should be done in a separate refactor for the serial lib.  Currently Yeti and uMWC for example still don't use it and have their own `openSerial` implementation